mk/external.mk

```diff
@@ -78,19 +89,31 @@ endef
 # Doom
 # https://tipsmake.com/how-to-run-doom-on-raspberry-pi-without-emulator
 DOOM_DATA_URL = http://www.doomworld.com/3ddownloads/ports/shareware_doom_iwad.zip
-DOOM_DATA = $(OUT)/DOOM1.WAD
+DOOM_DATA = $(DOOM_OUT)/DOOM1.WAD
```

Comment:
Can you make use of `eval` to let it more generic? That is, you may eliminate the use of `$(DOOM_OUT)`.

Reply:
This is unwanted code. Reverted.